2021-04-20 13:42:34,968 INFO    MainThread:2246 [wandb_setup.py:_flush():69] setting env: {}
2021-04-20 13:42:34,968 INFO    MainThread:2246 [wandb_setup.py:_flush():69] setting login settings: {}
2021-04-20 13:42:34,969 INFO    MainThread:2246 [wandb_init.py:_log_setup():336] Logging user logs to /opt/ml/code/src/wandb/run-20210420_134234-2tw9ysfm/logs/debug.log
2021-04-20 13:42:34,969 INFO    MainThread:2246 [wandb_init.py:_log_setup():337] Logging internal logs to /opt/ml/code/src/wandb/run-20210420_134234-2tw9ysfm/logs/debug-internal.log
2021-04-20 13:42:34,969 INFO    MainThread:2246 [wandb_init.py:init():369] calling init triggers
2021-04-20 13:42:34,969 INFO    MainThread:2246 [wandb_init.py:init():376] wandb.init called with sweep_config: {}
config: {}
2021-04-20 13:42:34,969 INFO    MainThread:2246 [wandb_init.py:init():418] starting backend
2021-04-20 13:42:34,969 INFO    MainThread:2246 [backend.py:_multiprocessing_setup():71] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-04-20 13:42:34,981 INFO    MainThread:2246 [backend.py:ensure_launched():123] starting backend process...
2021-04-20 13:42:34,986 INFO    MainThread:2246 [backend.py:ensure_launched():128] started backend process with pid: 2264
2021-04-20 13:42:34,987 INFO    MainThread:2246 [wandb_init.py:init():423] backend started and connected
2021-04-20 13:42:34,988 INFO    MainThread:2246 [wandb_init.py:init():465] updated telemetry
2021-04-20 13:42:34,989 INFO    MainThread:2246 [wandb_init.py:init():484] communicating current version
2021-04-20 13:42:36,235 INFO    MainThread:2246 [wandb_init.py:init():489] got version response upgrade_message: "wandb version 0.10.27 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2021-04-20 13:42:36,236 INFO    MainThread:2246 [wandb_init.py:init():497] communicating run to backend with 30 second timeout
2021-04-20 13:42:36,443 INFO    MainThread:2246 [wandb_init.py:init():522] starting run threads in backend
2021-04-20 13:42:36,469 INFO    MainThread:2246 [wandb_run.py:_console_start():1524] atexit reg
2021-04-20 13:42:36,469 INFO    MainThread:2246 [wandb_run.py:_redirect():1371] redirect: SettingsConsole.REDIRECT
2021-04-20 13:42:36,470 INFO    MainThread:2246 [wandb_run.py:_redirect():1376] Redirecting console.
2021-04-20 13:42:36,471 INFO    MainThread:2246 [wandb_run.py:_redirect():1438] Redirects installed.
2021-04-20 13:42:36,471 INFO    MainThread:2246 [wandb_init.py:init():546] run started, returning control to user process
2021-04-20 13:42:47,537 INFO    MainThread:2246 [wandb_run.py:_atexit_cleanup():1494] got exitcode: 1
2021-04-20 13:42:47,540 INFO    MainThread:2246 [wandb_run.py:_restore():1466] restore
2021-04-20 13:42:49,756 INFO    MainThread:2246 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 545
  total_bytes: 545
}

2021-04-20 13:42:50,459 INFO    MainThread:2246 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 545
  total_bytes: 3414
}

2021-04-20 13:42:50,860 INFO    MainThread:2246 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 3416
  total_bytes: 3416
}

2021-04-20 13:42:50,963 INFO    MainThread:2246 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 3416
  total_bytes: 3416
}

2021-04-20 13:42:51,067 INFO    MainThread:2246 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 3416
  total_bytes: 3416
}

2021-04-20 13:42:51,170 INFO    MainThread:2246 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 3416
  total_bytes: 3416
}

2021-04-20 13:42:51,273 INFO    MainThread:2246 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 3416
  total_bytes: 3416
}

2021-04-20 13:42:51,376 INFO    MainThread:2246 [wandb_run.py:_wait_for_finish():1616] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 3416
  total_bytes: 3416
}

2021-04-20 13:42:52,477 INFO    MainThread:2246 [wandb_run.py:_show_files():1838] logging synced files
